test/e2e/common.go

```diff
@@ -531,13 +531,7 @@ func dumpCRDS(ctx context.Context, cli client.Client, artifactFolder string) {
 }
 
 // DumpResources dumps resources related to BMO e2e tests as YAML.
-func DumpResources(ctx context.Context, e2eConfig *Config, clusterProxy framework.ClusterProxy, namespace string, artifactFolder string) {
-	// CAPI resources.
-	framework.DumpAllResources(ctx, framework.DumpAllResourcesInput{
-		Lister:    clusterProxy.GetClient(),
-		Namespace: namespace,
-		LogPath:   filepath.Join(artifactFolder, "clusters", clusterProxy.GetName(), "resources"),
-	})
```

Comment:
where do we define these stuffs now ike logPath for example @adilGhaffarDev ?

Reply:
We don’t need this, the `LogPath` is only used when dumping resources with the CAPI E2E framework. If we’re not planning to do that, there’s no need to set it. In this case, I don’t think we need to use the CAPI E2E framework at all.